HARK/ConsumptionSaving/ConsPortfolioModel.py

```diff
@@ -175,7 +175,7 @@ def update_solution_terminal(self):
         None
         """
         # Consume all market resources: c_T = m_T
-        cFuncAdj_terminal = IdentityFunction()
+        cFuncAdj_terminal = LinearInterp([0.0, 1.0], [0.0, 1.0])
         cFuncFxd_terminal = IdentityFunction(i_dim=0, n_dims=2)
```

Comment:
@llorracc this line and below use 2D IdentityFunction and ConstantFunction, should these be replaced as well?